tests/testthat/test-developer-gs_spending_bound.R

```diff
@@ -112,16 +118,25 @@ test_that("Two-sided asymmetric design fails to reproduce gsDesign test.type=3 b
 test_that("Two-sided asymmetric design fails to reproduce gsDesign test.type=4 bounds", {
   gsd <- gsDesign(test.type = 4, k = K, sfu = sfu, sfupar = sfupar, sfl = sfl, sflpar = sflpar, timing = timing,
                   delta = delta, alpha = alpha, beta = beta)
+  
   gsdv <- gs_design_npe(theta = delta, info = timing, beta = beta,
                         binding = FALSE, # Use this for test.type=4 and 6
                         upper = gs_spending_bound,
                         upar = list(sf = sfu, total_spend = alpha, param = sfupar),
                         lower = gs_spending_bound,
                         lpar = list(sf = sfl, total_spend = beta, param = sflpar))
   
+  # compare boundaries
   expect_equal(gsd$upper$bound, (gsdv %>% filter(Bound == "Upper"))$Z, tolerance = 7e-6)
   expect_equal(gsd$lower$bound, (gsdv %>% filter(Bound == "Lower"))$Z, tolerance = 9e-6)
-  expect_equal(gsd$n.I, (gsdv %>% filter(Bound == "Upper"))$info, tolerance = .04) # While tolerance should not be problematic, it seems large
+  
+  # compare statistical information
+  # While tolerance should not be problematic, it seems large
+  expect_equal(gsd$n.I, (gsdv %>% filter(Bound == "Upper"))$info, tolerance = .04) 
+  
+  # compare crossing boundaries probability under null hypothesis (theta = 0)
+  #expect_equal((gsdv %>% filter(Bound == "Upper"))$Probability0, sfu(alpha = alpha, t = timing, param = sfupar)$spend)
```

Comment:
Same.

Reply:
It is a pending test. I will revise it later in the test branch.